There was 2 typos at the very bottom of the page (#15955)
home-assistant/home-assistant.github.io: source/_docs/configuration/templating.markdown

```
{% endraw %}

> Would round `states('sensor.temperature')` to 2 decimal places, then divide `10` by that.

Reviewer: Actually, that is incorrect as well. 

As the filter operator `|` has priority, the `10` is rounded to 2 decimal places and the state value of `sensor.temperature` is casted/converted to a float type. Then the division takes place.

```suggestion
Would convert the state value of `sensor.temperatureround` to a float type, rounds `10` to 2 decimal places. As the final step, the float converted sensor value is divided by `10` (rounded to 2 decimals).
```

Author: Hello Frenck,  you are probably right but it is now beyond my understanding, the code example provided rounds 10 (the number?) to 2 decimal ? is there an interest to do so ? I'm a bit puzzled sorry, I though I had understood the example but now it makes no sense to me ^^

Reviewer: Well, I guess the thing this example tries to show, is that something is happening one would not expect: The `10` gets rounded.

All `|` operators are running before anything else first. Meaning the `10` gets rounded.

```
{% endraw %}

> Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00)
> This behavior is maybe not the one expected but priority rules implies that.

Reviewer: ```suggestion
Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00). This behavior is maybe not the one expected, but priority rules imply that.
```

> Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00)
> This behavior is maybe not the one expected but priority rules implies that.

Reviewer: ```suggestion
Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00). This behavior is maybe not the one expected, but priority rules imply that.

```
